pulsar-client-admin-api/src/main/java/org/apache/pulsar/client/admin/Transactions.java

```diff
@@ -135,9 +137,51 @@
      *
      * @return the metadata of slow transactions.
      */
-    CompletableFuture<Map<String, TransactionMetadata>> getSlowTransactions(long timeout,
-                                                                            TimeUnit timeUnit) throws Exception;
+    Map<String, TransactionMetadata> getSlowTransactions(long timeout, TimeUnit timeUnit) throws PulsarAdminException;
 
+    /**
+     * Get transaction coordinator internal stats.
+     *
+     * @param coordinatorId the coordinator ID
+     * @param metadata is get ledger metadata
+     *
+     * @return the future internal stats of this coordinator
+     */
+    CompletableFuture<CoordinatorInternalStats> getCoordinatorInternalStatsAsync(int coordinatorId, boolean metadata);
+
+    /**
+     * Get transaction coordinator internal stats.
+     *
+     * @param coordinatorId the coordinator id
+     * @param metadata whether to obtain ledger metadata
+     *
+     * @return the internal stats of this coordinator
+     */
+    CoordinatorInternalStats getCoordinatorInternalStats(int coordinatorId,
+                                                         boolean metadata) throws PulsarAdminException;
 
+    /**
+     * Get pending ack internal stats.
+     *
+     * @param topic the topic of get pending ack internal stats
+     * @param subName the subscription name of this pending ack
+     * @param metadata whether to obtain ledger metadata
+     *
+     * @return the future internal stats of pending ack
+     */
+    CompletableFuture<TransactionPendingAckInternalStats> getPendingAckInternalStatsAsync(String topic, String subName,
+                                                                                          boolean metadata);
+
+    /**
+     * Get pending ack internal stats.
+     *
+     * @param topic the topic of get pending ack internal stats
+     * @param subName the subscription name of this pending ack
+     * @param metadata whether to obtain ledger metadata
```

Comment:
```suggestion
     * @param metadata whether to obtain ledger metadata
```

double check: whether to obtain ledger metadata? or `* @param metadata the ledger metadata`